app/controllers/home-office-details.ts

```diff
@@ -19,24 +14,25 @@ function getHomeOfficeDetails(req: Request, res: Response, next: NextFunction) {
     next(e);
   }
 }
-
-async function postHomeOfficeDetails(req: Request, res: Response, next: NextFunction) {
-  try {
-    const homeOfficeDetails = req.body['homeOfficeRefNumber'];
-    const validation = homeOfficeNumberValidation(homeOfficeDetails);
-    if (validation) {
-      return res.render('appeal-application/home-office/details.njk',
-        {
-          error: validation
-        }
-      );
+function postHomeOfficeDetails(updateAppealService: UpdateAppealService) {
+  return async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const homeOfficeDetails = req.body['homeOfficeRefNumber'];
+      const validation = homeOfficeNumberValidation(homeOfficeDetails);
+      if (validation) {
```

Comment:
homeOfficeNumberValidation does return false but should we align like the others that return null ?